lib/pfconfig/namespaces/config/Switch.pm

```diff
@@ -46,6 +48,17 @@ sub build_child {
         SNMPCommunityTrap => 'public'
     };
 
+    my @management_ips;
+    push @management_ips, $self->{management_network}->tag('vip') if(defined($self->{management_network}->tag('vip')));
+    push @management_ips, $self->{management_network}->tag('ip') if(defined($self->{management_network}->tag('ip')));
+    foreach my $management_ip (@management_ips){
+        $tmp_cfg{$management_ip} = {
+            type => 'PacketFence', 
+            mode => 'production', 
+            radiusSecret => 'testing1234', 
+        };
+    }
+
```

Comment:
It makes it a valid switch as well so that if we want to run an eapol_test and go inside the PacketFence webservices call, we will be able too